<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AIPL-PUN-LT-081" ignored="1" name="Go Rest Regression Test" tests="8" failures="1" timestamp="06 Apr 2020 01:52:16 GMT" time="6.376" errors="0">
  <testcase name="createUserAPIPOSTTest" time="3.082" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="0.49" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="getUserListAPITest" time="5.166" classname="com.qa.gorest.tests.GetUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="0.51" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="deleteUserTest" time="5.647" classname="com.qa.gorest.tests.DeleteUserAPITest">
    <failure type="java.lang.AssertionError" message="expected [18] but found [24]">
      <![CDATA[java.lang.AssertionError: expected [18] but found [24]
at com.qa.gorest.tests.DeleteUserAPITest.deleteUserTest(DeleteUserAPITest.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserTest -->
  <testcase name="createUserAPIPOSTTest" time="0.41" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createANewUserAPIPUTTest_WithSetter" time="0.933" classname="com.qa.gorest.tests.UpdateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="0.414" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createANewUserAPIPUTTest_WithConstructor" time="0.0" classname="com.qa.gorest.tests.UpdateUserAPITest">
    <ignored/>
  </testcase> <!-- createANewUserAPIPUTTest_WithConstructor -->
</testsuite> <!-- Go Rest Regression Test -->
